schedule-pod-restart/README.md

```diff
@@ -0,0 +1,48 @@
+# Cronjob Pod Restart
+
+Automatically restarting pods with labels based on a cron schedule.
+
+## TL;DR
+
+```console
+helm repo add coditeu https://coditeu.github.io/charts
+helm repo update
+helm install coditeu/cronjob-pod-restart
+```
+
+## Introduction
+
+## Installing the Chart
+
+To install the chart with the release name `schedule-pod-restart`:
+
+```console
+$ helm install schedule-pod-restart coditeu/schedule-pod-restart
+```
+
+## Uninstalling the Chart
+
+To uninstall the `schedule-pod-restart` deployment:
+
+```console
+helm uninstall schedule-pod-restart
+```
+
+The command removes all the Kubernetes components associated with the chart and
+deletes the release.
+
+## Configuration
+
+The following table lists the configurable parameters of the chart and
+their default values.
+
+| Parameter                    | Description                                                              | Default                           |
+|:-----------------------------|:-------------------------------------------------------------------------|:----------------------------------|
+| `nameOverride`               | Name of the cronjob that will be created                                 | Chart Name `schedule-pod-restart` |
+| `serviceAccount.name`        | Name of the service account to use                                       | `operations-pod-restart`          |
+| `serviceAccount.annotations` | Annotations for the service account                                      | None                              |
+| `role.name`                  | Name of the role to create                                               | `manage-pods-role`                |
+| `bindings.role.name`         | Name of the role binding                                                 | `manage-pods-role-binding`        |
+| `cronjob.schedule`           | A cronjob expression that defines the schedule when the job should run   | `0 0 * * *`                       |
+| `cronjob.podLabels           | One or more labels on the pods that need to be restarted. Use format `key1=value1,key2=value2`      | None   |
+| `bitnamiContainerImageTag`   | bitnami/kubectl container image tag                                      | Version of the Kubernetes cluster where the chart will be deployed          |
```

Comment:
Maybe we should go with `image.tag` to align the default approach & not leak that we are using Bitnami?
```suggestion
| `image.tag`   | Tag of container image that contains `kubectl`                                      | Version of the Kubernetes cluster where the chart will be deployed          |
```

Reply:
Fixed in commit https://github.com/CoditEU/charts/pull/20/commits/807d90c57c6f579bacdf4b908af7e96a4a3a2d35. Note that I couldn't use "tag" (probably a reserved keyword), so changed it to "tagname" => image.tagname

Reply:
Thanks!